cirq/study/sweeps.py

```diff
@@ -101,6 +102,38 @@ def __iter__(self) -> Iterator[resolver.ParamResolver]:
         for params in self.param_tuples():
             yield resolver.ParamResolver(collections.OrderedDict(params))
 
+    # pylint: disable=function-redefined
+    @overload
+    def __getitem__(self, val: int) -> resolver.ParamResolver:
+        pass
+
+    # pylint: disable=function-redefined
```

Comment:
remove this and instead re-enable the rule after the `__getitem__` implementation.

Reply:
done.